federated_content_connector/signals.py

```diff
@@ -0,0 +1,27 @@
+"""federated_content_connector signals."""
+from logging import getLogger
+
+from federated_content_connector.models import CourseDetails
+from federated_content_connector.tasks import import_course_metadata
+
+LOGGER = getLogger(__name__)
+
+
+# TODO: Most probably this will be handled through an event on event bus
+def handle_courseoverview_import_course_details(sender, courserun_key):  # pylint: disable=unused-argument
```

Comment:
Note to Reviewers: Future of this handler is not certain and dependent on event bus approach as mentioned in https://twou.slack.com/archives/C049EGU04EQ/p1685551450300829